site/content/en/docs/reference/kubectl-kueue.md

```diff
@@ -75,17 +75,166 @@ You can run `kubectl kueue help` in the terminal to get the full list of command
 
 The following table includes short descriptions and the general syntax for all of the kueuectl operations:
 
-| Operation | Syntax                       | Description                                             |
-|--------|------------------------------|---------------------------------------------------------|
-| create | kubectl kueue create [TYPE] [NAME] [flags] | Create resource. | 
+| Operation | Syntax                                     | Description          |
+|-----------|--------------------------------------------|----------------------|
+| create    | kubectl kueue create [TYPE] [NAME] [flags] | Create a resource.   | 
+| resume    | kubectl kueue resume [TYPE] [NAME] [flags] | Resume the resource. | 
+| stop      | kubectl kueue stop [TYPE] [NAME] [flags]   | Stop the resource.   | 
 
 
-## Examples: Common operations
+## Resource types
 
-```shell
-# Create a local queue 
+The following table includes a list of all the supported resource types and their abbreviated aliases:
+
+| Name       | Short | API version            | Namespaced | Kind       |
+|------------|-------|------------------------|------------|------------|
+| localqueue | lq    | kueue.x-k8s.io/v1beta1 | true       | LocalQueue |
+| workload   | wl    | kueue.x-k8s.io/v1beta1 | true       | Workload   |
```

Comment:
Let's remove this for now, as it depends a lot on the command.

Reply:
Alternatively, you could have a list of supported types per command (or enhance the table above).

Reply:
Removed